doc/user/content/get-started/quickstart.md

```diff
@@ -306,6 +306,10 @@ DROP TABLE fraud_accounts;
 
 ## What's next?
 
+{{< note >}}
+For help getting started with your data or other questions about Materialize, you can schedule a [free guided trial](https://materialize.com/demo/?utm_campaign=General&utm_source=documentation).
```

Comment:
Nit, take it or leave it!

This sentence and the next seem to repeat themselves a bit the way they're currently structured. Maybe:

> To get started ingesting your own data from an external system like Kafka, MySQL or PostgreSQL, check the documentation for [sources](https://materialize.com/docs/sql/create-source/), and navigate to Data > Sources > New source in the [Materialize Console](https://console.materialize.com/) to create your first source.
>
> For help getting started with your data, or for other questions about Materialize, you can schedule a [free guided trial](https://materialize.com/demo/?utm_campaign=General&utm_source=documentation).